config/locales/en.controllers.yml

```diff
@@ -20,6 +20,9 @@ en:
         move:
           success: "%{count} transactions were reassigned."
           failure: "No transactions were reassigned. %{reassignment_error} on Order Detail #%{order_detail_id}."
+      thumbnail:
```

Comment:
I think these can go away since we're doing the upload/delete in the `Facility#update`